src/plugins/intel_gna/gna_graph_compiler.cpp

```diff
@@ -2068,12 +2068,19 @@ case name:\
                 output_pwl_scale_factor,
                 gnaFlags->input_low_precision);
         } else {
+            auto is_fused_with_conv_2d = false;
+            auto prevDnnLayer = dnnComponents.findComponent(prevLayer);
+            if (prevDnnLayer && prevDnnLayer->operation == kDnnConvolutional2dOp) {
+                is_fused_with_conv_2d = true;
+            }
+
             PwlDesignOpt(activation_type,
```

Comment:
Shouldn't we cover `PwlDesign` too (uniformPwlDesign)?

Reply:
Inserting of extra segments is currently added only to the PwlDesignOpt().
Should I add this also to the PwlDesign()?

Reply:
It was discussed with @nkogteva and @dorloff and decision is that adding limitation for extra segments of errata will be updated only for the place in code when errata was applied until now.

uniformPwlDesign option is not clear and is suppose to be cleared as deprecated. There will be created separate cleanup ticket created  for that by @dorloff.

Reply:
I am fine with that.